front_end/panels/rn_welcome/rn_welcome-legacy-meta.ts

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
I can't name this file `rn_welcome-meta-legacy.ts` because of this:
https://github.com/facebookexperimental/rn-chrome-devtools-frontend/blob/1467870f834c61c6f00ce2577c41bb159d2d1ef8/tsconfig.json#L18-L20